### ISSUE TITLE
Github Posthook

### ISSUE THREAD

**Issue by tkanos** [tkanos/twx#21]
We need to create a meta posthook command to help to send the twtxt file to git

**Comment by bkil** [comment]
Good idea! Allowing for as many gratis hosting providers as possible would be nice. See also:
* https://docs.gitlab.com/ee/api/
* https://docs.gitea.io/en-us/api-usage/
* https://developer.atlassian.com/server/bitbucket/reference/rest-api/
* https://en.wikipedia.org/wiki/WebDAV (NextCloud, etc.)
